go.sum

```diff
@@ -266,8 +266,6 @@ github.com/clusterhq/flocker-go v0.0.0-20160920122132-2b8b7259d313/go.mod h1:P1w
 github.com/cncf/udpa/go v0.0.0-20191209042840-269d4d468f6f/go.mod h1:M8M6+tZqaGXZJjfX53e64911xZQV5JYwmTeXPW+k8Sc=
 github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa/go.mod h1:zn76sxSg3SzpJ0PPJaLDCu+Bu0Lg3sKTORVIj19EIF8=
 github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
-github.com/codefresh-io/go-sdk v0.34.5 h1:9ZPVd1bZgExRRXfttNd0XqB9CIWdDK6qRSQLpjLXIWY=
-github.com/codefresh-io/go-sdk v0.34.5/go.mod h1:CcoVmTFWHGkbrSW8LyOGB/vJe5Vzr3iC/pNE2QIBTyg=
```

Comment:
after go-sdk will be merged, I will run go mod tidy again, and then (if I'm not wrong) I will not get the error I'm getting now when running that command:
go: github.com/codefresh-io/go-sdk@v0.34.6: reading github.com/codefresh-io/go-sdk/go.mod at revision v0.34.6: unknown revision v0.34.6